### PR TITLE
Make WSL2 installation instructions more detailed

### PR DESCRIPTION
**Pull request by janekb04**
Add some missing steps in the WSL2 installation process and describe it more throughoutly.

### CODE REVIEW

File: docs/windows.md [hunk 1]
```diff
@@ -1,32 +1,82 @@
...
+            (New-Object System.Net.WebClient).DownloadFile("https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi", "C:/Windows/Temp/wsl2.msi");
+            msiexec /i "C:/Windows/Temp/wsl2.msi";
+            Remove-Item "C:/Windows/Temp/wsl2.msi"
+            ```
+        - Set WSL2 as default WSL version
+            ```powershell
+            wsl --set-default-version 2
+            ```
+    - Install a Linux distribution from the Microsoft Store (cannot be done using command line, 
+      unless using something like Chocolatey, but in that case the whole process is automated):
```
Comment (nvasilakis) [inline docs/windows.md:59]: Here we can only include distributions for which we have verified that PaSh runs (currently these are Ubuntu 18.04 and 20.04, Fedora, Debian, and Arch.)

**Review by nvasilakis — APPROVED** [review]
Thanks so much for this, @janekb04! Feel free to merge (using squash) after you make the changes:-)

**Comment by janekb04** [comment]
@nvasilakis I don't think I have the privileges to merge right now. I also won't need to squash, as the aggregators will be in a separate PR to keep things clean.

**Comment by nvasilakis** [comment]
Ah, thanks for letting me know @janekb04 , I did not know you don't have the privileges. I will take care of merging then:-)

Would it be easy to [address the comment in my review](https://github.com/binpash/pash/pull/275#discussion_r668086895)? I will then merge, thanks so much!

File: docs/windows.md (hunk 1 above)
Comment (janekb04) [reply docs/windows.md:59]: Ok, I removed the untested distros. Sorry, I didn't notice your comment here, but GitHub only notified me about your top-level comment.

Comment (nvasilakis) [reply docs/windows.md:59]: No worries, and thank you!

**Comment by angelhof** [comment]
@janekb04 @nvasilakis I think we should not copy WSL installation instructions in PaSh's documentation. A link to the official instructions is better (since otherwise we would have to maintain a copy of WSL installation instruction making sure that it doesn't become stale).

**Comment by nvasilakis** [comment]
@angelhof agreed! Documentation is getting a serious overhaul, will fix accordingly!

**Pull request merged**